client/src/components/Related/RelatedProducts.jsx

```diff
@@ -1,20 +1,50 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
```

Comment:
This is the container component for my related products - it maps the products from the relatedProducts state in Relatex.jsx and adds the data from each product to a ProductsCard.jsx component